### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by MarkoHiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# KEEPR example responses
+# KEEPR example api responses for liveticker and matchday
 
-Use the example json files to simulate a full fledged API response. This helps app developers design the views.
+Use the example json files to simulate a full fledged API response. These json files should help app developers to design the views.
+For each case (matchday and liveticker) a whitelist should be usedin the app in order to be able to ignore unknown events.
```
Comment (jojahner) [inline README.md]: Is this Better?
"For each case (matchday and liveticker) the app should maintain a event whitelist to handle unknown events (e.g. by not showing them)."

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# KEEPR example responses
+# KEEPR example api responses for liveticker and matchday
 
-Use the example json files to simulate a full fledged API response. This helps app developers design the views.
+Use the example json files to simulate a full fledged API response. These json files should help app developers to design the views.
```
Comment (jojahner) [inline README.md]: "to design _their_ views." ?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
-# KEEPR example responses
+# KEEPR example api responses for liveticker and matchday
```
Comment (johanness) [inline README.md]: api or API?

Comment (jojahner) [inline README.md]: YO API DIGGA?

**Pull request merged**